README.md

```diff
@@ -6,11 +6,9 @@
 
 ## Overview
 
-Luigi is a micro frontend JavaScript framework that enables you to create an administrative user interface driven by local and distributed views. Luigi allows a web application to communicate with the micro frontends which the application contains. To make sure the communication runs smoothly, you can easily configure the settings such as routing, navigation, authorization, and user experience elements.
+[Luigi](https://luigi-project.io) is a micro frontend JavaScript framework that enables you to create an administrative user interface driven by local and distributed views. Luigi allows a web application to communicate with the micro frontends which the application contains. To make sure the communication runs smoothly, you can easily configure the settings such as routing, navigation, authorization, and user experience elements.
 
-Luigi consists of Luigi Core and Luigi Client libraries. They interact with each other to leverage communication between the core application and the micro frontend, without compromising the security principles behind the iframe pattern.
-
-Go to the [Luigi project website](https://luigi-project.io/) to learn more about the product.
+Luigi consists of Luigi Core (main application) and Luigi Client (micro frontend) libraries which communicate securely through postMessage API.
```

Comment:
I was thinking if we can get rid of the brackets here - we usually don't use them. I'm not saying it's wrong, but maybe there is a way to rework the sentence a bit.